plugins/talk-plugin-google-auth/client/actions.js

```diff
@@ -0,0 +1,7 @@
+export const loginWithGoogle = () => (dispatch, _, { rest }) => {
+  window.open(
+    `${rest.uri}/auth/google`,
+    'Continue with Google',
```

Comment:
This should be a translated string.

Reply:
oh yeah, good catch. yeah, it's basically cut and paste. I'll update the patch today.

Reply:
Looking into this further, the windowName parameter of window.open is only used for references to the window (like <a target="ref">), it's not actually the window title, so therefore it doesn't make sense to translate it. To reduce future confusion, I think it might be best to set the string to something like 'google-login-flow', what do you all think?